src/table/TableWrapper.jsx

```diff
@@ -31,10 +33,14 @@ export default function TableWrapper(props) {
   const [page, setPage] = useState(0);
   const [rowsPerPage, setRowsPerPage] = useState(100);
   const focusedCellCoord = useRef([0, 0]);
+  const shouldRefocus = useRef(false);
   const classes = useStyles();
   const containerMode = constraints.active ? 'containerOverflowHidden' : 'containerOverflowAuto';
   const paginationHidden = constraints.active && 'paginationHidden';
   const paginationFixedRpp = selectionsAPI.isModal() || tableWidth < 400;
+  const setShouldRefocus = () => {
+    shouldRefocus.current = rootElement.getElementsByTagName('table')[0].contains(document.activeElement);
```

Comment:
unit test does not cover here.

Reply:
Testing this function specifically is tricky. It's nice to have it here since all the needed variables are in this scope. I guess I can rewrite it so it runs the logic inside `handle-cell-focus` instead.